integration/integration_test.go

```diff
@@ -63,6 +64,10 @@ func init() {
 		check.Suite(&ProxyProtocolSuite{})
 		check.Suite(&Etcd3Suite{})
 	}
+
+	if *osio {
+		check.Suite(&OSIOSuite{})
+	}
```

Comment:
New flag 'osio' added to just run OSIO related integration tests.  Need to introduce new flag as currently test for container flag is failing due to addition of OSIO middleware.